woocommerce_fusion/tasks/sync_sales_orders.py

```diff
@@ -629,6 +629,10 @@ def create_missing_items(self, items_list, woocommerce_site):
 				row.woocommerce_id = cstr(item_data.get("product_id"))
 				row.woocommerce_server = woocommerce_site
 				item.flags.ignore_mandatory = True
+				item.supplier_items = [
```

Comment:
This is not the field we are interested in. We are interested in `item.item_defaults.default_supplier`. Note that `item_defaults` is usually set by default unlike `supplier_items` that is empty by default